migtests/scripts/jenkins_wrapper

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+export LD_LIBRARY_PATH=${LDLIBRARYPATH}
+export TARGET_DB_HOST=${TARGET_DB_HOST}
+export TARGET_DB_PORT=${TARGET_DB_PORT}
+export TARGET_DB_USER=${TARGET_DB_USER}
+export TARGET_DB_PASSWORD=${TARGET_DB_PASSWORD}
+test_name=$1
+
+bash ./migtests/scripts/run-test.sh "${test_name}"
```

Comment:
Just forward all arguments as it is to the child bash (https://stackoverflow.com/questions/3811345/how-to-pass-all-arguments-passed-to-my-bash-script-to-a-function-of-mine)

```
bash "$@"
```
This will also allow you to use the `jenkins_wrapper` script to run "import data file" tests (which are not driven by the `run-test.sh`).